__integrations__/environments/env.base

```diff
@@ -26,3 +26,4 @@ SHOW_LOGS=false
 # Test Data
 # ----------
 
+BEARER_AUTH_HEADER=x-pagopa-lollipop-auth
```

Comment:
```suggestion
BEARER_AUTH_HEADER=x-pagopa-lollipop-auth-jwt
```
shouldn't this be `x-pagopa-lollipop-auth-jwt`?

Reply:
We defined it `x-pagopa-lollipop-auth` in [openapi param](https://github.com/pagopa/io-functions-lollipop/blob/558f8899b905bbbaa828b566726de81a156d8513/openapi/external.yaml#L65) and [config](https://github.com/pagopa/io-functions-lollipop/blob/558f8899b905bbbaa828b566726de81a156d8513/utils/config.ts#L74)